packages/esm-appointments-app/src/calendar/appointments-calendar-view.component.tsx

```diff
@@ -1,29 +1,31 @@
-import React, { useState } from 'react';
+import React, { useEffect } from 'react';
 import dayjs from 'dayjs';
 import { useTranslation } from 'react-i18next';
 import { useAppointmentsCalendar } from '../hooks/useAppointmentsCalendar';
-import type { CalendarType } from '../types';
 import AppointmentsHeader from '../header/appointments-header.component';
 import CalendarHeader from './header/calendar-header.component';
-import CalendarView from './calendar-view.component';
-import { useAppointmentDate } from '../helpers';
+import { changeSelectedDate, useSelectedDate } from '../helpers';
+import MonthlyCalendarView from './monthly/monthly-calendar-view.component';
+import { useParams } from 'react-router-dom';
 
 const AppointmentsCalendarView: React.FC = () => {
   const { t } = useTranslation();
-  const [calendarView, setCalendarView] = useState<CalendarType>('monthly');
-  const { currentAppointmentDate, setCurrentAppointmentDate } = useAppointmentDate();
-  const { calendarEvents } = useAppointmentsCalendar(dayjs(currentAppointmentDate).toISOString(), calendarView);
+  const { selectedDate } = useSelectedDate();
+  const { calendarEvents } = useAppointmentsCalendar(dayjs(selectedDate).toISOString(), 'monthly');
+
+  let params = useParams();
```

Comment:
```suggestion
  const { date: selectedDate } = useParams();
```

With the appropriate accompanying changes is probably cleaner?